examples/bank_account/bank_account.py

```diff
@@ -26,7 +26,7 @@ def balance(self):
     def withdraw(self, amount):
         """ withdraw money """
         if self._balance >= amount > 0:
-            self._balance -= (amount + self._calc_commission_rate(self._hasCommissionDiscount))
+            self._balance += (amount + self._calc_commission_rate(self._hasCommissionDiscount))
```

Comment:
The withdraw method should subtract the amount from the balance, not add it. Please revert the change. [important]